### DISCUSSION TITLE
In the Ballista and Turret information window, indicate the damage range for the Ballista and for each Turret

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#2783] [Ideas]
In the Ballista and Turret information window, indicate the damage range for the Ballista and for each Turret, at the end of the text.

Example:
"The Ballista... ...attack skill (Damage: 20-40)."
"The Left Turret... ...attack skill (Damage: 10-20)."
"The Right Turret... ...attack skill (Damage: 10-20)."

In fheroes2:

![Ballista fh2 01](https://user-images.githubusercontent.com/43583994/107701520-5540ed00-6cb9-11eb-919b-4355dbde4ac7.png)

![Ballista fh2 02](https://user-images.githubusercontent.com/43583994/107701533-58d47400-6cb9-11eb-80a4-8b6d99a78c1a.png)

**Comment by Branikolog** [comment]
I think this should be optional.
Maybe, we should create a "newbie assist" mode, which would include this proposal... As well as numeric description of neutral creature stacks on adventure map... And probably something else...

**Reply by LeHerosInconnu** [reply]
@Branikolog.

> I think this should be optional.
> Maybe, we should create a "newbie assist" mode, which would include this proposal... As well as numeric description of neutral creature stacks on adventure map... And probably something else...

Personally, I am not a newcomer to this game. :)
And I'm going to use it to decide which troops to recruit in defense for a castle (before a combat).
And which troops to target based on the potential damage during a combat.

**Reply by Branikolog** [reply]
I think, you never forget the damage range of towers while siege. ;) So is it really necessary? This window has too much information inside, so I not sure we should add more.

**Reply by LeHerosInconnu** [reply]
@Branikolog.

> I think, you never forget the damage range of towers while siege. ;) So is it really necessary? This window has too much information inside, so I not sure we should add more.

You know that the damage of the Ballista and the Turrets changes according to the attack skill of the Captain/Hero, right?
You know that it is possible to control dozens of castles, right?

**Reply by Branikolog** [reply]
Correct me, if I'm wrong, but the damage also affected by the defense if a target? 
So, each time turret shoots different target, we get different damage? 
So, anyway damage is predictable, but you cannot tell exact value...

**Reply by LeHerosInconnu** [reply]
@Branikolog.

> Correct me, if I'm wrong, but the damage also affected by the defense if a target?
> So, each time turret shoots different target, we get different damage?
> So, anyway damage is predictable, but you cannot tell exact value...

I would like to have the base of the damage range to make the decision to shoot a troop that will be more or less affected by this damage.
This can be useful during combat.
If the base of the damage range is for example (Damage: 30-60), I know that it's interesting to target a troop whose creatures have a lot of hp (Bone Dragon, Cyclops, etc.) or that it's possible to quickly get rid of a troop of a few Grand Elves.
If the base of the damage range is for example (Damage: 10-20), I know that it's less interesting to target a troop whose creatures have a lot of hp and that I can concentrate the fire on troops with less hp to do maximum damage (since it will be difficult or impossible to annihilate a Black Dragon in say a 5 turns combat).
Similarly, before a combat, when I plan the defense of a castle based on a nearby hero who might attack the castle the next turn, this base of the damage range allows me to adjust the amount of creatures to recruit knowing that if the Balistta and Turrets damage is high enough, it won't be necessary to recruit many creatures.

**Comment by LeHerosInconnu** [comment]
The damage range displayed should correspond to the archer creature's base damage multiplied by the number of archers.

In fheroes2:

![Archer info window 01](https://github.com/ihhub/fheroes2/assets/43583994/c34ec31d-4202-41de-930c-18dbd8ee0df1)

For example, with 10 archers for the ballista, and 5 archers for each turret:
- 10 x 2 and 10 x 3, i.e. "20-30"
- 5 x 2 and 5 x 3, i.e. "10-15"

So, for the text:
"The Ballista... ...attack skill (Damage: 20-30)."
"The Left Turret... ...attack skill (Damage: 10-15)."
"The Right Turret... ...attack skill (Damage: 10-15)."

Or perhaps, for greater precision:
"The Ballista... ...Archers (Damage: 20-30)... ...attack skill."
"The Left Turret... ...Archers (Damage: 10-15)... ...attack skill."
"The Right Turret... ...Archers (Damage: 10-15)... ...attack skill."

The sentences could also be simplified:
"The Ballista fires with the strength of 10 Archers (damage: 20-30) with 1 to attack skill."
"The Left Turret fires with the strength of 5 Archers (damage: 10-15) with 1 to attack skill."
"The Right  fires with the strength of 5 Archers (damage: 10-15) with 1 to attack skill."

I think the word "The" could also be removed for a less cumbersome wording:
"Ballista fires with the strength of 10 Archers (damage: 20-30) with 1 to attack skill."
"Left Turret fires with the strength of 5 Archers (damage: 10-15) with 1 to attack skill."
"Right  fires with the strength of 5 Archers (damage: 10-15) with 1 to attack skill."

**Reply by zenseii** [reply]
I've also been thinking that this whole text could be cut in half by combining the left and right turrets' texts -> "The Left and Right Turrets fire with....". They will always fire with the same strength if they're both built anyways.

**Reply by LeHerosInconnu** [reply]
Hello @zenseii,

Yes, but in the course of combat it can happen, quite often in fact, that one of the turrets is destroyed and not the other.

In fheroes2:

![One turret destroyed 01](https://github.com/ihhub/fheroes2/assets/43583994/ea2dd729-8fa5-4b85-b623-427bc8332cb4)

**Comment by LeHerosInconnu** [comment]
This is planned in my mod: https://github.com/ihhub/fheroes2/discussions/10112.
I close the discussion.
